vantage/cost_report_resource.go

```diff
@@ -239,6 +243,7 @@ func (r CostReportResource) Read(ctx context.Context, req resource.ReadRequest,
 	state.ChartType = types.StringValue(out.Payload.ChartType)
 	state.DateBin = types.StringValue(out.Payload.DateBin)
 	state.WorkspaceToken = types.StringValue(out.Payload.WorkspaceToken)
+	state.FolderToken = types.StringValue(out.Payload.FolderToken)
```

Comment:
Not sure why this was missing. Was causing inconsistent behavior on import, would require an apply.